parsec/core/gui/workspaces_widget.py

```diff
@@ -244,10 +244,21 @@ def goto_file_clicked(self):
         if button is not None:
             try:
                 path = button.workspace_fs.decrypt_file_link_path(addr)
+                ts = button.workspace_fs.decrypt_timestamp(addr)
+
+                # Mount workspace at the link's creation time
+                if ts is not None:
```

Comment:
And if we have no timestamp the workspace isn't mounted?

Reply:
Done.